src/r_sky.c

```diff
@@ -42,255 +43,394 @@ boolean stretchsky;
 //
 // sky mapping
 //
-
 int skyflatnum;
-int skytexture = -1; // [crispy] initialize
-int skytexturemid;
-
-sky_t *sky = NULL;
+sky_t *levelskies = NULL;
+static skydefs_t *skydefs;
+static fixed_t defaultskytexturemid;
 
 // PSX fire sky, description: https://fabiensanglard.net/doom_fire_psx/
+// [EA] done away with original spreadFire, in favor of Odamex's texture-
+// replacing implementation.
+static void SpreadFire(int src, byte* fire, int width)
+{
+  const byte pixel = fire[src];
+  const int copyloc0 = src - width;
+  if(pixel == 0)
+  {
+    if (copyloc0 >= 0)
+      fire[copyloc0] = 0;
+  }
```

Comment:
Please reformat the new functions to match the style of the rest of the file: https://github.com/chocolate-doom/chocolate-doom/blob/master/HACKING.md
The formatting can be automated using the `clang-format` tool.

Reply:
Basically `clang-format -i src/*sky*` :wink: